function_lab.js

```diff
@@ -0,0 +1,263 @@
+// # Function Exercises
+//
+// ## Question 1
+//
+// Write a function that returns the largest of two numbers. If they are equal, return the second one.
+//
+// Example:
+//
+// Input: `12, 4`
+//
+// Output: `12`
+//
+
+                 //parameter refereing to num1 and num 2
+const largest =(num1, num2) => { // function is searching for the largest varibales
+  if(num1 < num2){ // first requirement num1 < num2
+  return num1 // print result num1
+} if(num1 > num2){ // closing for if with curly brackets // 2req. num1 >num2
+  return num1 // print result num1
+} else if(num1 === num2) { // at last if the 3req. is true
+   return num2 // print num2
+ } // closing with curly brackets the 3 ifstat.
+}
+const resultOfLargest = largest(12,4)
+console.log(resultOfLargest) // print in the value you want the function to use
```

Comment:
Love how you write everything out breaking it down. 

the first return according to what we want and your condition should actually return num2
we have to remember correct syntax ; we need an else if before the next condition